packages/vc-handler-ld-local/src/ld-credential-module.ts

```diff
@@ -147,17 +182,30 @@ export class LdCredentialModule {
     //AssertionProofPurpose()
   ): Promise<boolean> {
     // console.log(JSON.stringify(presentation, null, 2))
-
-    const result = await vc.verify({
-      presentation,
-      suite: this.getAllVerificationSuites(),
-      documentLoader: this.ldDocumentLoader.getLoader(context, fetchRemoteContexts),
-      challenge,
-      domain,
-      presentationPurpose,
-      compactProof: false,
-      checkStatus,
-    })
+    let result
+    if (presentation.proof.type === 'BbsBlsSignature2020') {
+      //Should never be null or undefined
```

Comment:
to review

Reply:
Not the same as vc.verify https://github.com/digitalcredentials/vc-js/blob/9e40e67a3060289c899890c2cfb76681cb893368/lib/vc.js#L397

Reply:
Cannot use it because vc-js uses different packages

require('@digitalcredentials/jsonld');
const jsigs = require('@digitalcredentials/jsonld-signatures');